packages/default/scss/messagebox/_layout.scss

```diff
@@ -0,0 +1,14 @@
+@include exports("message-box/layout") {
+
+    // Base
+    .k-message-box {
+        margin: $message-box-margin;
```

Comment:
There are some visual changes in the message box in the chat- there is additional margin at the bottom. I suppose this should be fixed